packages/terra-dev-site/src/webpack/plugin/siteLoaderUtils.js

```diff
@@ -1,5 +1,6 @@
 const rehypeSlug = require('rehype-slug');
 const rehypeUrl = require('rehype-urls');
+const rehypeUrlInspector = require('@jsdevtools/rehype-url-inspector');
```

Comment:
we use this plugin because it also gives you back the file and location of the link.